### ISSUE TITLE
Gotify Notifications problem

### ISSUE THREAD

**Issue by Albenss** [containrrr/watchtower#719]
**Describe the bug**
Watchtower self-updated about an hour ago, and the Gotify service doesn't work anymore.

**To Reproduce**
With my following docker-compose.yml and no modification:
```
watchtower:
    image: containrrr/watchtower:latest
    container_name: Watchtower
    restart: always
    network_mode: host
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - /etc/timezone:/etc/timezone:ro
      - /etc/localtime:/etc/localtime:ro
      - $CONFIG_PATH/watchtower/config.json:/config.json
    environment:
      - "TZ=$TZ"
      - "WATCHTOWER_CLEANUP=true"
      - "WATCHTOWER_LABEL_ENABLE=true"
      - "WATCHTOWER_POLL_INTERVAL=15000"
      - "WATCHTOWER_NOTIFICATIONS=gotify"
      - "WATCHTOWER_NOTIFICATION_GOTIFY_URL=$GOTIFY_URL"
      - "WATCHTOWER_NOTIFICATION_GOTIFY_TOKEN=$GOTIFY_TOKEN"
    labels:
      - "com.centurylinklabs.watchtower.enable=true"
```

**Logs from running watchtower with the `--debug` option**
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x835877]
goroutine 1 [running]:
github.com/spf13/viper.(*Viper).find(0x0, 0xc45c5e, 0x17, 0x17, 0x1)
	/go/pkg/mod/github.com/spf13/viper@v1.4.0/viper.go:979 +0x37
github.com/spf13/viper.(*Viper).Get(0x0, 0xc45c5e, 0x17, 0xc0000aaab0, 0xc3f796)
	/go/pkg/mod/github.com/spf13/viper@v1.4.0/viper.go:671 +0x7c
github.com/spf13/viper.(*Viper).GetString(0x0, 0xc45c5e, 0x17, 0x0, 0xd)
	/go/pkg/mod/github.com/spf13/viper@v1.4.0/viper.go:734 +0x3f
github.com/containrrr/watchtower/pkg/notifications.newGotifyNotifier(0x1133ba0, 0x10dfde4, 0x5, 0x5, 0x1, 0x1)
	/home/circleci/repo/pkg/notifications/gotify.go:31 +0x7c
github.com/containrrr/watchtower/pkg/notifications.NewNotifier(0x1133ba0, 0xd2e7a0)
	/home/circleci/repo/pkg/notifications/notifier.go:43 +0x3e5
github.com/containrrr/watchtower/cmd.PreRun(0x1133ba0, 0x1172f58, 0x0, 0x0)
	/home/circleci/repo/cmd/root.go:111 +0x397
github.com/spf13/cobra.(*Command).execute(0x1133ba0, 0xc0000301d0, 0x0, 0x0, 0x1133ba0, 0xc0000301d0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:755 +0x4f9
github.com/spf13/cobra.(*Command).ExecuteC(0x1133ba0, 0x406325, 0xc000102058, 0x0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fe
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/containrrr/watchtower/cmd.Execute()
	/home/circleci/repo/cmd/root.go:51 +0x31
main.main()
	/home/circleci/repo/main.go:13 +0x25
```

**Comment by simonaronsson** [comment]
Hi!

Thank you for your report! This was likely caused by #684 which wasn't meant to be released until `v1.2.0`. I've reverted that commit and released `v1.1.3`, which should reach you in an hour or so. After that, your config should (hopefully) work as before.

Best,
Simme

**Comment by Albenss** [comment]
Hello, yes, after a new pull of the Watchtower image, the gotify service works as expected, thanks!
